templates/portfolios/manage_portfolio_modal.html

```diff
@@ -0,0 +1,35 @@
+<div id="manage_portfolio_modal" class="modal" tabindex="-1" role="dialog" aria-labelledby="manage_portfolio_modal_title" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title" id="manage_portfolio_modal_title">Select a Portfolio</h4>
+      </div>
+      <form role="form" method="POST">
+          {% csrf_token %}
+        <div class="modal-body">
+          <label for="id_portfolio">{{ project_form.portfolio.label }}</label>:&nbsp;
+          {{ project_form.portfolio }}
+         </div>
+         <div class="modal-footer">
+           <a id="edit_portfolio_submit" class="btn btn-success btn-submit" type="submit">Edit</a>
+           <a id="delete_portfolio_submit"  class="btn btn-danger btn-submit confirm-delete" type="submit" onclick="return confirm('Are you sure you want to delete this?')">Delete Portfolio</a>
```

Comment:
I like this method for giving a confirmation message. I had done something much more complicated for one of my tickets, but this is nice and clean.